src/base/utils/misc.cpp

```diff
@@ -250,13 +250,20 @@ QString Utils::Misc::friendlyUnit(const qint64 bytesValue, const bool isSpeed)
            + unitString(unit, isSpeed);
 }
 
-int Utils::Misc::friendlyUnitPrecision(SizeUnit unit)
+int Utils::Misc::friendlyUnitPrecision(const SizeUnit unit)
 {
     // friendlyUnit's number of digits after the decimal point
-    if (unit == SizeUnit::Byte) return 0;
-    if (unit <= SizeUnit::MebiByte) return 1;
-    if (unit == SizeUnit::GibiByte) return 2;
-    return 3;
+    switch (unit) {
```

Comment:
What's the point of this change? Isn't it subjective?

Reply:
Generally `switch` is as good as `if-else`: https://stackoverflow.com/questions/97987/advantage-of-switch-over-if-else-statement
Although it is less significant for small cases, I still don't want to miss the chance to use here. So it's more like code style unification.